exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -345,6 +348,31 @@ void createBlockWithTransactions() {
     }
   }
 
+  @Test
+  void findTransactionsInPool() {
```

Comment:
Shall we also add a more focused test (because of #879) verifying that submitted through Node transactions appear in a pool (this one still needed to test findTransactionsInPool, but is more complex/won't provide as precise diagnostics in case native testkit or core breaks)?

Reply:
Added a test for that